crates/services/block_aggregator_api/src/api.rs

```diff
@@ -65,4 +75,12 @@ impl<T> BlockAggregatorQuery<T> {
         let query = Self::GetCurrentHeight { response: sender };
         (query, receiver)
     }
+
+    pub fn new_block_subscription()
+    -> (Self, tokio::sync::mpsc::Receiver<(BlockHeight, Block)>) {
+        const ARBITRARY_CHANNEL_SIZE: usize = 10;
```

Comment:
this should probably be larger, because it is `mpsc`, and if the consumer is slow, it will block the producer

Reply:
Yes. These constructors are just used in tests for now. I'll add a `#[cfg(test)]` to imply the consumer is responsible for defining the channel size.